frameworks/Rust/hyper/src/multiple_queries.rs

```diff
@@ -0,0 +1,59 @@
+use std::convert::Infallible;
+
+use http::header::{CONTENT_LENGTH, CONTENT_TYPE, SERVER};
+use http::Response;
+use http_body_util::combinators::BoxBody;
+use http_body_util::{BodyExt, Full};
+use hyper::body::Bytes;
+use serde::Serialize;
+use tokio_postgres::Row;
+
+use crate::db::POOL;
+use crate::{Error, Result, APPLICATION_JSON, SERVER_HEADER};
+
+const QUERY: &str = "SELECT id, randomnumber FROM world WHERE id = $1";
+
+#[derive(Debug, Serialize)]
+pub struct World {
+    id: i32,
+    randomnumber: i32,
+}
+
+impl From<Row> for World {
+    fn from(row: Row) -> Self {
+        World {
+            id: row.get(0),
+            randomnumber: row.get(1),
+        }
+    }
+}
```

Comment:
This is duplicated in the two db tests. I'm mostly fine with that here as it makes the tests more self-contained when reading them for understanding, and allows for future tweaks if the tests diverge.